html2docx/html2docx.py

```diff
@@ -183,8 +204,17 @@ def handle_starttag(self, tag: str, attrs: List[Tuple[str, Optional[str]]]) -> N
             self.init_run([("underline", True)])
         elif tag == "ul":
             self.add_list_style("List Bullet")
+        elif tag == "table":
+            self.init_table(attrs)
+        elif tag == "tr":
+            self.table_data.append([])
 
     def handle_data(self, data: str) -> None:
+        if self.tag in ("td", "th"):
+            if self.table_data:
+                self.table_data[-1].append(data)
```

Comment:
The HTML is expected to be valid, `th` and `td` must be in a table context.
```suggestion
            self.table_data[-1].append(data)
```